src/components/Tagline/Tagline.tsx

```diff
@@ -0,0 +1,12 @@
+"use client";
+import styles from "./Tagline.module.scss";
+
+export function Tagline() {
+  return (
+    <main className={styles.main}>
```

Comment:
ideally for a website like this, we'd have a global css file with our h1, h2, h3 defined. That way for future sections like this, we could just use h2 or h3 here. it makes it easier to adjust our fonts later by just having them defined in one spot